en/docs/install-and-setup/setting-up-servers.md

```diff
@@ -280,7 +280,7 @@ can be applied using a single configuration file, which is the `deployment.toml`
 
     - Go to the `<APIM_HOME>/<OB_APIM_TOOLKIT_HOME>/repository/resources` directory.
     
-    - Rename `wso2am-4.0.0-deployment-cds.toml` to `deployment.toml`.
+    - Based on the API Manager version you have, rename `wso2am-4.1.0-deployment-cds.toml` or `wso2am-4.0.0-deployment-cds.toml` to `deployment.toml`.
```

Comment:
Is there a file named `wso2am-4.1.0-deployment-cds.toml`?

Reply:
Removed the toml file.